services/web/server/src/simcore_service_webserver/statics.py

```diff
@@ -28,29 +26,103 @@
     FRONTEND_APP_DEFAULT,
     FRONTEND_APPS_AVAILABLE,
     FrontEndAppSettings,
+    StaticsSettings,
 )
 
 STATIC_DIRNAMES = FRONTEND_APPS_AVAILABLE | {"resource", "transpiled"}
 
-APP_FRONTEND_BASEDIR_KEY = f"{__name__}.frontend_basedir"
-APP_STATICS_OUTDIR_KEY = f"{__file__}.outdir"
+APP_FRONTEND_CACHED_INDEXES_KEY = f"{__file__}.cached_indexes"
+APP_FRONTEND_CACHED_STATICS_JSON_KEY = f"{__file__}.cached_statics_json"
 
+# NOTE: saved as a separate item to config
+STATIC_WEBSERVER_SETTINGS_KEY = f"{__name__}.StaticWebserverModuleSettings"
 
 log = logging.getLogger(__file__)
 
 
-@lru_cache()
-def get_index_body(statics_outdir: Path, frontend_name: str):
-    index_path = statics_outdir / frontend_name / "index.html"
-    html = index_path.read_text()
-    # TODO: this is not very safe ...
-    # fixes relative paths
-    html = html.replace(f"../resource/{frontend_name}", f"resource/{frontend_name}")
-    html = html.replace("boot.js", f"{frontend_name}/boot.js")
-    return html
+class StaticWebserverModuleSettings(BaseSettings):
```

Comment:
STYLE: please move to ``statics_settings.py`` together with other statics-related settings.